Header header
float32 joint_angle
float32 servo1_position
float32 servo2_position
float32 u1
float32 u2
float32 u3

